lib/generators/landlord/install_generator.rb

```diff
@@ -1,18 +1,20 @@
-require 'rails/generators/active_record'
+require "rails/generators"
+require "rails/generators/active_record"
 
 module Landlord
   module Generators
-    class InstallGenerator < ActiveRecord::Generators::Base
-      source_root File.expand_path('templates', __dir__)
+    class InstallGenerator < Rails::Generators::Base
+      include ActiveRecord::Generators::Migration
+
+      source_root File.expand_path("templates", __dir__)
 
       def copy_migration_files
-        migration_template 'create_tenants.rb', 'db/migrate/create_tenants.rb'
-        migration_template 'create_tenant_connection.rb', 'db/migrate/create_tenant_connection.rb'
+        migration_template "create_tenants.rb", "db/migrate/create_tenants.rb"
+        migration_template "create_tenant_connections.rb", "db/migrate/create_tenant_connections.rb"
       end
 
-      def generate_models
-        copy_file 'tenant.rb', 'app/models/tenant.rb'
-        copy_file 'tenant_connection.rb', 'app/models/tenant_connection.rb'
```

Comment:
these are classes that come with the gem, we don't really need them inside the app. So I made an engine out of this